src/plugins/condition/components/CurrentOutput.vue

```diff
@@ -32,11 +32,11 @@
             @click="expanded = !expanded"
         ></span>
     </div>
-    <div v-if="expanded && condition"
+    <div v-if="expanded && output"
```

Comment:
According to Charles there was an error in the UX and current output section should always be expanded. There shouldn't be a show/hide arrow in title bar.

Reply:
should also remove the expanded prop and not pass it to component. This component becomes so simple I wonder if it's needed at all.

Reply:
@JoelMcKinnon Please create a new issue for this.

Reply:
Done. #2700